view/frontend/layout/catalog_product_view.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
+    <body>
+        <referenceBlock name="product.info.addtocart">
+            <block class="Bolt\Boltpay\Block\JsProductPage" name="bolt.button" template="Bolt_Boltpay::button_product_page.phtml" before="-" />
+        </referenceBlock>
+        <referenceBlock name="product.info.addtocart.additional">
+            <block class="Bolt\Boltpay\Block\JsProductPage" name="bolt.button.additional" template="Bolt_Boltpay::button_product_page.phtml" before="-" />
+        </referenceBlock>
```

Comment:
What's the reason to add the block twice? Adding 2 buttons?

Reply:
@danac-gs First block works for simple and grouped products and the second one for configurable and downloadable.
Nothing works for bundle products so we need to add the third block for this type of product if we need it down the road.